### ISSUE TITLE
Unable to connect to 10.0.2.0/24 IP Addresses

### ISSUE THREAD

**Issue by jcderr** [docker/machine#1742]
When using docker-machine, my docker-machine vm (and thus any containers therein) are unable to connect to hosts on my VLAN with 10.0.2.0/24 addresses. This occurs because `eth0` is configured on this subnet and does not appear to be modifiable.

    docker@default:~$ ifconfig eth0
    eth0      Link encap:Ethernet  HWaddr 08:00:27:8D:BA:49
              inet addr:10.0.2.15  Bcast:10.0.2.255  Mask:255.255.255.0
              inet6 addr: fe80::a00:27ff:fe8d:ba49/64 Scope:Link
              UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
              RX packets:245973 errors:0 dropped:0 overruns:0 frame:0
              TX packets:54958 errors:0 dropped:0 overruns:0 carrier:0
              collisions:0 txqueuelen:1000
              RX bytes:277515200 (264.6 MiB)  TX bytes:3500591 (3.3 MiB)
    docker@default:~$ curl -I http://10.0.106.250:8080
    HTTP/1.1 200 OK
    Content-Type: application/json
    Date: Thu, 20 Aug 2015 15:00:08 GMT
    Content-Length: 202

    docker@default:~$ curl -I http://10.0.2.224:8000
    curl: (7) Failed connect to 10.0.2.224:8000; No route to host
    docker@default:~$ exit
    jcderr@JCDerrs-MBP:~$ curl -I http://10.0.2.224:8000
    HTTP/1.1 200 OK
    Server: SimpleHTTP/0.6 Python/2.7.6
    Date: Thu, 20 Aug 2015 15:01:23 GMT
    Content-type: text/html; charset=UTF-8
    Content-Length: 3722
    
    jcderr@JCDerrs-MBP:~$

**Comment by ehazlett** [comment]
Yeah I think we will probably run into similar issues the more we work with VPNs etc.
